viz/src/machines/viz.typegen.ts

```diff
@@ -0,0 +1,40 @@
+// This file was automatically generated. Edits will be overwritten
+
+export interface Typegen0 {
```

Comment:
Is this a new feature from xstate typegen ?

Reply:
What are you talking about precisely?

Reply:
The file itself, I don't remember being creating such a file before :o
My memory is playing tricks on me

Reply:
Time goes too fast. Typegen has always worked by generating a `.typegen.ts` file.
But an interesting feature has landed in VSCode recently, which allows to hide `.typegen.ts` files: https://code.visualstudio.com/updates/v1_67#_explorer-file-nesting.